wcfsetup/install/files/lib/acp/form/PackageEnableUpgradeOverrideForm.class.php

```diff
@@ -99,41 +103,213 @@ private function isEnabled()
 
     private function getIssuesPreventingUpgrade()
     {
-        $issues = [];
+        $parameters = ['issues' => []];
+        EventHandler::getInstance()->fireAction($this, 'getIssuesPreventingUpgrade', $parameters);
 
-        $objectTypes = ObjectTypeCache::getInstance()->getObjectTypes('com.woltlab.wcf.searchableObjectType');
-        $tableNames = [];
-        foreach ($objectTypes as $objectType) {
-            $tableNames[] = SearchIndexManager::getTableName($objectType->objectType);
+        $issues = [
+            ...$parameters['issues'],
```

Comment:
The app checks should possibly be moved to the bottom for a more logical order of “Core stuff first”.